assembler/index.js

```diff
@@ -55,8 +54,6 @@ module.exports = async function cdnAssembler(config, targetBucket, opts) {
     let sourceDir = path.join(workDir, 'sources');
     let assembledDir = path.join(workDir, 'assembled');
 
-    const messages = initMessager(opts);
```

Comment:
You removed this, but it still references the variable

Reply:
It is being used as well, so it will probably cause errors

Reply:
Good catch! I fixed it